docs/kratos/social-signin/81_lark.mdx

```diff
@@ -0,0 +1,86 @@
+---
+id: lark
+title: Add lark as a social sign-in provider in Ory
+sidebar_label: Lark
+---
+
+# Lark
+
+Follow these steps to add Lark as a social sign-in provider to your project using the Ory CLI:
+
+1. Create an application in [Feishu Open Platform Developer](https://open.feishu.cn/app) Background and obtain the ID and secret
```

Comment:
What does the word "Background" refer to?

Reply:
Sorry, it is a typo.

It should be `Create an application in Developer Console...` and the `Developer Console` is a web app used to configure the OAuth client.